hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/BlockOutputStreamEntryPool.java

```diff
@@ -117,7 +119,8 @@ public BlockOutputStreamEntryPool(
   }
 
   ExcludeList createExcludeList() {
-    return new ExcludeList();
+    return new ExcludeList(getConfig().getExcludeNodesExpiryTime(),
+        Clock.system(ZoneOffset.UTC));
```

Comment:
new use createExcludeList() for constructor BlockOutputStreamEntryPool() at line 140 as used.
https://github.com/apache/ozone/pull/5530/files#diff-db8e4914806b73e8ec37c15ffd146fc70bd1fcba631df9a314c7389af64a895dL140

Reply:
yeah updated.